src/ServiceControlInstaller.Engine/Validation/QueueNameValidator.cs

```diff
@@ -112,17 +118,23 @@ internal void CheckQueueNamesAreUniqueWithinInstance()
         internal void CheckQueueNamesAreNotTakenByAnotherAuditInstance()
         {
             var allQueues = AuditInstances.SelectMany(instance => new QueueNameValidator(instance).queues);
-            CheckQueueNamesAreNotTaken(allQueues);
+            var duplicates = (
+                from queue in queues
+                where allQueues.Any(p =>
+                    p.QueueType != QueueType.Audit &&
```

Comment:
I'm not sure that this quite captures every case. What if `queue` is not an audit queue but `p` is?

i.e. 

- First instance has:
  - Audit queue `audit`
  - Audit log queue `audit.log`
- Second instance has:
  - Audit queue `audit2`
  - Audit log queue `audit`

Reply:
@mikeminutillo 	within the context of this method:

- The 'queues' will always be from SCA, so either an Audit queue or Audit Log queueu.
- The method `CheckQueueNamesAreNotTakenByAnotherAuditInstance` itself is just for checking audit queues. The check for ServiceControl queues is a different one `CheckQueueNamesAreNotTakenByAnotherServiceControlInstance` and that goes over `SCInstances`.

So I think the `queue` within this method is always either audit or audit log queue. The check compares the name (Audit + Audit Log) against all the other queues (everything "Audit" queues) of other instances installed.